BridgeSDK/BridgeAPI/SBBAuthManager.m

```diff
@@ -296,14 +297,19 @@ - (NSString *)sessionToken
     }
 }
 
-- (NSURLSessionTask *)signUpWithJSON:(NSDictionary *)json completion:(SBBNetworkManagerCompletionBlock)completion
-{
-    return [_networkManager post:kSBBAuthSignUpAPI headers:nil parameters:json completion:completion];
-}
-
 - (NSURLSessionTask *)signUpStudyParticipant:(SBBSignUp *)signUp completion:(SBBNetworkManagerCompletionBlock)completion
 {
-    NSMutableDictionary *json = [[signUp dictionaryRepresentation] mutableCopy];
+    // If there's a placeholder StudyParticipant object, it may be partially filled-in, so let's start with that
+    NSString *studyParticipantType = SBBStudyParticipant.entityName;
+    SBBStudyParticipant *studyParticipant = (SBBStudyParticipant *)[SBBComponent(SBBCacheManager) cachedSingletonObjectOfType:studyParticipantType createIfMissing:NO];
+    NSMutableDictionary *json = [NSMutableDictionary dictionary];
+    if (studyParticipant) {
+        json = [[SBBComponent(SBBObjectManager) bridgeJSONFromObject:studyParticipant] mutableCopy];
+    }
+    
+    // now overwrite with entries from signUp object
+    NSDictionary *signUpJson = [signUp dictionaryRepresentation];
+    [json addEntriesFromDictionary:signUpJson];
```

Comment:
Participant info is cleared from the cache upon signOut, and the auth delegate method is called with nil for the userSessionInfo.